src/test/regress/expected/multi_modifying_xacts.out

```diff
@@ -1173,6 +1173,7 @@ SELECT create_distributed_table('numbers_hash_failure_test', 'key');
 -- and rename the existing user
 \c - :default_user - :worker_1_port
 ALTER USER test_user RENAME TO test_user_new;
+NOTICE:  Citus partially supports ALTER ROLE for distributed databases
```

Comment:
Not very clear what "partially support" means from this context.

Reply:
yeah, I agree!

This warning was there already but is visible now that the feature is on by default. I believe it was added because renaming the roles was the _only_ `ALTER ROLE` command that was not implemented. I believe the `HINT` or `DETAIL` of this messages gives extra information to the user.

Maybe we should just bite the bullet and implement `RENAME` as well. No reason to have it not implemented with this vague message.

Reply:
made the message more specific, I wonder if we should only give this error for distributed users though, or go even as far as just implementing it. But will defer that to a separate change